resources/startup.sh

```diff
@@ -138,7 +138,7 @@ function runMain() {
     sql "INSERT INTO settings (name, value, updated_on) VALUES ('ui_theme','Cloudogu', now());"
 
     # we use markdown as default format, however it can be changed
-    sql "INSERT INTO settings (name, value, updated_on) VALUES ('text_formatting', 'markdown', now());"
+    sql "INSERT INTO settings (name, value, updated_on) VALUES ('text_formatting', 'common_mark', now());"
```

Comment:
Very nice. 

What we also need is a post-upgrade script that migrates the old formatter if it was set to `markdown`. Users should not make these things manually when we can do it with ease for them.

Reply:
Super nice!

The ending `fi` looks crooked though in terms of indentation